util/feeding.py

```diff
@@ -31,8 +31,9 @@ def read_csvs(csv_files):
 
 
 def samples_to_mfccs(samples, sample_rate, train_phase=False, wav_filename=None):
-    if train_phase and sample_rate != FLAGS.audio_sample_rate:
-        tf.print('WARNING: sample rate of file', wav_filename, '(', sample_rate, ') does not match FLAGS.audio_sample_rate. This can lead to incorrect results.')
+    if train_phase:
+        sample_rate_flag = tf.constant(FLAGS.audio_sample_rate, name='FLAGS.audio_sample_rate')
+        tf.cond(tf.math.not_equal(sample_rate, sample_rate_flag), lambda: tf.print('WARNING: sample rate of file', wav_filename, '(', sample_rate, ') does not match FLAGS.audio_sample_rate. This can lead to incorrect results.'), lambda: tf.ones([0], dtype=tf.dtypes.bool), name='matching_sample_rate')
```

Comment:
nit: please break the indentation into lines, you can use `tf.no_op()` in the else clause, and you don't need to explicitly create a `tf.constant`.
```suggestion
    if train_phase:
        tf.cond(tf.math.not_equal(sample_rate, FLAGS.audio_sample_rate),
                lambda: tf.print('WARNING: sample rate of file', wav_filename, '(', sample_rate, ') does not match FLAGS.audio_sample_rate. This can lead to incorrect results.'),
                lambda: tf.no_op(),
                name='matching_sample_rate')
```

Reply:
Don't feel sorry, I should have seen it as well.